python/cloud-devrel-kokoro-resources/python-base/Dockerfile

```diff
@@ -118,7 +114,8 @@ RUN for PYTHON_VERSION in 2.7.18 3.9.17; do \
   && rm -rf ~/.cache/ \
   && rm -r "$GNUPGHOME"
 
-# Make sure the Python 3 version of pip is up-to-date and install virtualenv
+# Make sure the Python 3 version of pip is up-to-date and install
+# python packages
 COPY requirements.txt /requirements.txt
 
 RUN /usr/local/bin/python3 -m pip install --upgrade pip
```

Comment:
Please also test each python version like we do in
https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.kokoro/docker/Dockerfile#L166